Project.toml

```diff
@@ -7,10 +7,36 @@ version = "0.6.1"
 DataStructures = "864edb3b-99cc-5e75-8d2d-829cb0a9cfe8"
 Dates = "ade2ca70-3891-5945-98fb-dc099432e06a"
 Downloads = "f43a241f-c20a-4ad4-852c-f6b1247861c6"
-Requires = "ae029012-a4dd-5104-9daa-d747884805df"
+NetCDF = "30363a11-5582-574a-97bb-aa9a979735b9"
```

Comment:
Do we need NetCDF and Zarr as hard dependencies here?

Reply:
I don't think so. I will have a look.

Reply:
The test also go through without Zarr and NetCDF as hard dependencies. Strangely enough, they didn't show up in my local environment. 🤷🏼